src/Sinch/Fax/FaxClient.cs

```diff
@@ -17,20 +18,26 @@ public interface ISinchFax
 
         /// <inheritdoc cref="ISinchFaxEmails" />
         public ISinchFaxEmails Emails { get; }
+
+        /// <inheritdoc cref="ISinchFaxServices " />
+        public ISinchFaxServices Services { get; }
     }
 
     internal class FaxClient : ISinchFax
     {
         internal FaxClient(string projectId, Uri baseAddress, LoggerFactory? loggerFactory, IHttp http)
         {
             Faxes = new FaxesClient(projectId, baseAddress, loggerFactory?.Create<ISinchFaxFaxes>(), http);
-            Emails = new EmailsClient(projectId, baseAddress, loggerFactory?.Create<ISinchFaxEmails>(), http);
+            Services = new ServicesClient(projectId, baseAddress, loggerFactory?.Create<ISinchFaxServices>(), http);
+            Emails = new EmailsClient(projectId, baseAddress, loggerFactory?.Create<ISinchFaxEmails>(), http, Services);
         }
 
         /// <inheritdoc />
         public ISinchFaxFaxes Faxes { get; }
 
         /// <inheritdoc />
         public ISinchFaxEmails Emails { get; }
+
+        public ISinchFaxServices Services { get; }
```

Comment:
Other class members have the comment `/// <inheritdoc />` above them. Can you confirm it is not needed for this one?